scripts/microshift-sos-report.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_NAME=$(basename "$0")
+PROFILES="network,security,microshift"
+PLUGINS="container_log,crio,rpmostree,rpm"
+TEMPDIR="/tmp"
+
+function usage() {
+    echo "Collect an sos report including all relevant information for MicroShift"
+    echo "Includes"
+    echo "  profiles: ${PROFILES}"
+    echo "  plugins: ${PLUGINS}"
+    echo ""
+    echo "Usage: ${SCRIPT_NAME} [--tmp-dir TMP-DIR]"
+    echo "   --tmp-dir TMP-DIR  Temporary directory for saving the report. Defaults to ${TEMPDIR}."
```

Comment:
Should we add a command line option for specifying plugins by user?

Reply:
The script is mainly a one line command only, I thought of it as a helper for non-advanced users. If you need to customize the command it looks easier to me to just run `sos report` directly instead, wdyt?